src/test/java/org/apache/ibatis/migration/hook/testdir/hooks/after_new.js

```diff
@@ -0,0 +1,19 @@
+/*
+ *    Copyright 2010-2018 the original author or authors.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+if (typeof println == 'undefined')
+  this.println = print;
+println("after new change supplied " + (change !== null))
```

Comment:
Now that I look at this, the script isn't comprehensive enough. I need to test for the existence of `paths` and `args` since those are passed in with the bindings